lib/Varien/Io/File.php

```diff
@@ -247,11 +251,12 @@ public function streamClose()
         if (!$this->_streamHandler) {
             return false;
         }
-
         if ($this->_streamLocked) {
             $this->streamUnlock();
         }
-        @fclose($this->_streamHandler);
+        if (is_resource($this->_streamHandler)) {
```

Comment:
From #2040, this is not good, there is a better way with `$this->_IsValidSource()`.